### ISSUE TITLE
How to apply pronunciation rules to ARIA attributes?

### ISSUE THREAD

**Issue by minorninth** [w3c/pronunciation#87]
Sorry if I missed something, but I'd like to better understand how pronunciation rules could be added to an ARIA attribute that's normally read out by screen readers, such as aria-label, aria-roledescription, aria-valuetext, etc.

Right now the spec seems to focus on pronunciation rules for static text in an HTML document only, but for a modern web page that makes use of ARIA, it seems important to understand how those pronunciation rules should apply to accessible text in ARIA attributes too.

**Comment by AutoSponge** [comment]
This is an excellent point. Thank you for the comment. It's still being reviewed by the TF. I think the use of `aria-labelledby` would surpass `aria-label` where speech information were needed. But this is a bigger issues for `aria-valuetext` which doesn't have an idref corollary.
